src/WaterPokemon.java

```diff
@@ -0,0 +1,43 @@
+package src;
+
+public class WaterPokemon extends Pokemon{
+
+    //VARIABLES
+    private String defence;
+    private String special;
+
+
+    //CONSTRUCTOR
+    public WaterPokemon(String name, int level, int hp, String defence, String special) {
+        super(name, level, hp);
+        this.defence = defence;
+        this.special = special;
+    }
+
+
+    //GET & SET
+    public String getDefence() {
+        return defence;
+    }
+
+    public void setDefence(String defence) {
+        this.defence = defence;
+    }
+
+    public String getSpecial() {
+        return special;
+    }
+
+    public void setSpecial(String special) {
+        this.special = special;
+    }
+
+
+    //METHOD
+    public void attacks(String special) {
+        System.out.println(super.getName() + " attacks you with " + special);
```

Comment:
Goed gebruik gemaakt van super, maar waarom geef je hier `String special` mee in de parameter terwijl je al een `String special` als instance variable op regel 7 hebt staan?